bitnami/jaeger/templates/_helpers.tpl

```diff
@@ -0,0 +1,68 @@
+{{/* vim: set filetype=mustache: */}}
+
+
+{{/*
+Return the proper jaeger&trade; image name
+*/}}
+{{- define "jaeger.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper Docker Image Registry Secret Names
+*/}}
+{{- define "jaeger.imagePullSecrets" -}}
+{{ include "common.images.pullSecrets" (dict "images" (list .Values.image .Values.volumePermissions.image .Values.backup.uploadProviders.google.image .Values.backup.uploadProviders.azure.image) "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Create the name of the query deployment
+*/}}
+{{- define "jaeger.query.fullname" -}}
+    {{ printf "%s-query" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the name of the collector deployment. This name includes 2 hyphens due to
+an issue about env vars collision with the chart name when the release name is set to just 'jaeger'
+ref. https://github.com/jaegertracing/jaeger-operator/issues/1158
+*/}}
+{{- define "jaeger.collector.fullname" -}}
+    {{ printf "%s-collector" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create the name of the agent deployment
+*/}}
+{{- define "jaeger.agent.fullname" -}}
+    {{ printf "%s--agent" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Return the cassandra subchart password.
```

Comment:
There are helpers not used anywhere

```
helper jaeger.cassandraSubChartPassword defined is not used anywhere in the yaml
helper jaeger.imagePullSecrets defined is not used anywhere in the yaml
helper jaeger.initdbScriptsConfigmapName defined is not used anywhere in the yaml
helper jaeger.initdbScriptsSecret defined is not used anywhere in the yaml
```

Reply:
Done [ba41a9a](https://github.com/bitnami/charts/pull/13480/commits/ba41a9a98c493cbfdc289c8d2ada2d3576571b6f)